kong/plugins/session/storage/kong.lua

```diff
@@ -91,15 +91,15 @@ function kong_storage:insert_session(sid, data, expires)
   }, { ttl = self.lifetime })
 
   if err then
-    ngx.log(ngx.ERR, "Error inserting session: ", err)
+    kong.log.err("Error inserting session: ", err)
```

Comment:
Just a note, not a blocker for this PR:

1. At runtime (in Kong's proxy and Admin API, for instance), we never capitalize error messages
2. We typically phrase errors like so: `"could not do X:", err`, or `"failed to do X:", err`. Mentioning "Error" here creates duplication in the log which already has an `[error]` prefix indicating its log level.

Reply:
oh good note, makes sense